src/content/experimental/creation-flows/creation-flows.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Creation flows
+---
+
+### Creating is the generation of a new resource. Relocating an existing resource is adding, not creating.
+
+**Status**: Experimental
+
+**Maintainers**: [Vikki Paterson](https://github.ibm.com/VICTORIA), [Arnaud Gillard](https://github.ibm.com/Arnaud-Gillard)
+
+<AnchorLinks>
+
+- [Standard creation](#standard-creation)
+- [Multistep creation](#multistep-creation)
+- [Confirmation](#confirmation)
+
+</AnchorLinks>
+
+<ImageComponent caption="Example of a creation pattern in context." cols="8">
+
+![Example](images/1.png)
+
+</ImageComponent>
+
+## Standard creation
+
+The standard creation pattern is used when no stepped guidance is needed and the input fields are directly related to each other. 
+
+<AnchorLinks small>
+
+- [Modal](#modal)
+- [Side panel](#side-panel)
+- [Full screen](#full-screen)
+- [Advanced option](#advanced-option)
+
+</AnchorLinks>
+
+### Modal
+
+If there are only one or two decisions to be made during a creation, use a small, standard modal overlaying the content where the creation was initiated. Avoid scrolling inside a small modal. 
+
+<ImageComponent caption="Example of a standard creation modal" cols="8">
+
+![Example of a standard creation modal](images/2.png)
+
+</ImageComponent>
+
+### Side panel
+
+If you have limited screen real estate, consider using a side panel. Side panels also keep the user aware of the context in which they were working. You may choose to use either the slide in or slide over panels.
+
+<ImageComponent caption="Example of a side panel creation flow" cols="8">
+
+![Example of a side panel creation flow](images/3.png)
+
+</ImageComponent>
+
+### Full page
+
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full page option. Full page creation flows allow you to take advantage of maximum screen real estate.
+
+<ImageComponent caption="Example of a full page creation flow" cols="8">
+
+![Example of a fill page creation flow](images/8.png)
```

Comment:
I think I would arrange this page more like this:
- Unless buttons are attached to the edge of a container (image doesn't count) then the buttons should have a 8px space between them. 
- They should also be aligned with the title at top. Base align the text of the title with the text of the button.
- Could you also fix the hanging e in "workspace"
![image](https://user-images.githubusercontent.com/11670886/59884362-b34eed00-937d-11e9-9f8b-3b335e711fcd.png)